backend/cms/models/page.py

```diff
@@ -143,7 +143,7 @@ def get_tree(cls, region_slug, archived=False):
     def best_language_title(self):
         page_translation = self.page_translations.filter(language__code=get_language())
         if not page_translation:
-            alt_code = LanguageTreeNode.objects.get(region__id=self.region.id).get_root().language.code
+            alt_code = LanguageTreeNode.objects.filter(region__id=self.region.id).first().get_root().language.code
```

Comment:
Sure this works, but in this case the shortcut `alt_code = self.region.default_language.code` would be even better.

Reply:
Yeah, I've just seen this in ulli's pr. :laughing: